### ISSUE TITLE
primary-group: is not taken...

### ISSUE THREAD

**Issue by CloudeLecaw** [lucko/LuckPerms#2691]
inside my player files i have the setting

primary-group: admin

But the player does not get the permission of admin

This only happen if i add admin to the parents...
parents:
- admin
- player

i thought parents are the thing other called inheritance:.... 

this is since the migration of the permissionsex database
LuckPerms_5.1.93

**Comment by underscore11code** [comment]
I'm not sure quite what you're saying. If you're trying to say setting the primary group to something doesn't take affect unless you add the group to parents, yes, that is how it should be.

By default you shouldn't need to touch the primary-group, it should be automatically calculated based off of weight.

**Comment by CloudeLecaw** [comment]
i tried aggain with the latest version same issue

ok so:
i migrated the dabase from permissionsex to LuckPerms

This is how my playerfile looks after --(i chose the yaml storeage)

```
uuid: f24df990-9c6b-4f32-938a-4c91a26d31f4
name: Cloude_Lecaw
primary-group: admin
permissions:
- gadgetsmenu.suits.frog.*
parents:
- spieler
```

now when i log in im still counting as player until i add the group admin to the parents

**Comment by underscore11code** [comment]
Yes, that's correct behavior*. If a user doesn't inherit from a group, it can't be their primary group. As stated in my above comment, the primary-group should be automatically calculated without admin intervention.

*If the first example is what our `/lp migration` spat out, then perhaps there's a bug in the PEX migration. That's a different issue.

tl;dr this is working as expected.

**Comment by CloudeLecaw** [comment]
i tried it aggain. 
Now all worked fine and all staff players got teir right groups added
Well thank you for helping with support <3
